posthog/models/cohort.py

```diff
@@ -145,9 +145,11 @@ def calculate_people(self, use_clickhouse=is_clickhouse_enabled()):
     def calculate_people_ch(self):
         if is_clickhouse_enabled():
             from ee.clickhouse.models.cohort import recalculate_cohortpeople
+            from posthog.tasks.calculate_cohort import calculate_cohort
 
             try:
                 recalculate_cohortpeople(self)
+                calculate_cohort.delay(self.id)
```

Comment:
Since this already runs inside a Celery task, I propose to `calculate_cohort` directly, without `delay`, as currently `calculate_people_ch` would succeed without an error even if `calculate_cohort` does fail.

Reply:
yeah fair enough